packages/components/src/tooltip/test/index.js

```diff
@@ -4,7 +4,10 @@
 import { shallow, mount } from 'enzyme';
 import TestUtils from 'react-dom/test-utils';
 import ReactDOM from 'react-dom';
-
+/**
+ * Internal dependencies
+ */
+import { TOOLTIP_DELAY } from '../index.js';
```

Comment:
Do we need to export and import this? can't we define one for the test alone?

Reply:
I think it is better to use the same constant in both tests and the code itself as it is is a constant

Reply:
does this by being exported become a part of the public API?

Reply:
it is not exported in components/src/index.js so I believe it is not public.